swagger.json

```diff
@@ -5850,6 +6001,72 @@
             "items": {
                 "$ref": "#/definitions/DocumentType"
             }
+        },
+        "User": {
+            "type": "object",
+            "properties": {
+                "id": {
+                    "type": "integer",
+                    "description": "The id of the User record.",
+                    "example": 123
+                },
+                "name" : {
+                    "type": "string",
+                    "description": "The first and last name of the user.",
+                    "example": "Jane Doe"
+                },
+                "email": {
+                    "type": "string",
```

Comment:
We can define `"format": "email"` as a hint of the format type, even though it isn't supported in swagger: https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
I like this idea, but it throws a ton of errors in the swagger editor preview.

Reply:
👍